input/pagecontent/authorization.md

```diff
@@ -1,4 +1,186 @@
+
+
+#### Authoritative data sources
+
+**Trust Service Provider Certificates**: 
+PKIoverheid certificates. Used to identify the organization operating the client or server of data user or data holder. 
+
+**CiBG-DeZI**: 
+- Used to identify the practitioner and care provider
+- Role of the Practitioner. Uses (indirectly) the BIG-register as data source
+
+**CiBG-LRZa**:
+- Registry of care providers and their care service directory endpoint-urls
+
+**Care service directories**:
+- Can data holder handle my request? 
+- Endpoint-url of data holder
+
+**SBV-Z**:
+- Used to check the identity (BSN) of the patient
+
+**Vektis Organization type**:
+- Vektis is the source of the care provider type. A care provider can be one or more types. For example: a pharmacy, hospital, general practitioner, care at home, etc.
+
+**VZVZ Mitz**:
+- Patient Consent preference registry
+
+**NictiZ Qualifications**:
+- Is data user (care provider and software) qualified for the request? This source claims the data user to be abled to use some CapabilityStatements from NictiZ Healthcare Information standards. The request should be in scope of the CapabilityStatements.
+
+**NictiZ Healthcare Information standards**:
+Healthcare Information standards define which operations a data user or data holder should use/support. These standards also define HealthCare Information Models (HCIM's) and therefore which data labels, codes or categories can be used in policies. 
+
+#### Autorization policy makers
+Authorization policies SHALL be expressed in the [Rego policy language](https://www.openpolicyagent.org/docs/policy-language) to avoid semantic ambiguity and support automated testing.
+<!-- publishing policies, policies should be merged without negotiation... -->
+
+#### Requesting party (Data user)
+Creates a request, adds necessary attributes/claims about itself and sends it to the responding party 
+
+#### Responding party (Data holder)
+Acquires attributes/claims about requesting party and e.g. the patient consent. Evaluates the request.
+May have recorded resource-level access for data user while sending notification (TA Notified Pull mechanism)
+Implementers are not required to use the Rego-policy-language (nor the OpenID AuthZen API) in production systems, but the outcome of their authorization decisions SHALL match the outcome using the original, specified authorization policy. 
+
+
+### Data models
+GF Authorization uses the information model specification of [OpenID AuthZen](https://openid.net/specs/authorization-api-1_0.html). The information model for requests and responses include the following entities: Subject, Action, Resource, Context, and Decision. The use of this information model is illustrated by an example of a data user searching for active MedicationRequests for a patient.
+
+```
+{
+  "subject": {
```

Comment:
I think this would be literally be the token introspection result, with registered claims